release-please-config.json

```
@@ -0,0 +1,15 @@
+{
+  "bootstrap-sha": "6c543a5930c519c3fac807377b400ef423a6d312",
+  "release-type": "terraform-module",
+  "packages": {
+    ".": {
+      "release-type": "terraform-module",
+      "draft": false,
+      "prerelease": false,
+      "bumpMinorPreMajor": false,
```

```suggestion
      "bump-minor-pre-major": false,
```
And this is already the default value